test/unit/transforms/util/githubApiRequests.test.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import GitHubAPI from "../../../../src/config/github-api";
+import { compareCommitsBetweenBaseAndHeadBranches } from "../../../../src/transforms/util/githubApiRequests";
+import { getLogger } from "../../../../src/config/logger";
+
+describe("GitHub API Request Suite", () => {
+	describe("compareCommitsBetweenBaseAndHeadBranches", () => {
+		it("should return message from multiple commits containing multiple issue keys", async () => {
+			const workflowRunPayload = Object.assign(
+				{},
+				require("../../../fixtures/workflow-basic.json")
+			);
+
+			const { pull_requests, repository } =
+				workflowRunPayload.payload.workflow_run;
+
+			const payload = {
+				owner: repository.owner.login,
+				repo: repository.name,
+				base: pull_requests[0].base.ref,
+				head: pull_requests[0].head.ref,
+			};
+
+			const pullRequestCommits = Object.assign(
+				{},
+				require("../../../fixtures/api/pull-request-multiple-commits-diff.json")
+			);
+
+			const data = pullRequestCommits.data;
```

Comment:
nit: consistency between destructuring above/in other files and here
`const { data } = pullRequestCommits;`